scripts/start.sh

```diff
@@ -80,6 +85,6 @@ default:
 EOL
 
 printf "\e[0;32m*****STARTING SERVER*****\e[0m\n"
-echo "bash -c '${STARTCOMMAND[*]}'"
-su steam -c "bash -c '${STARTCOMMAND[*]}'"
+echo "bash -c '${STARTCOMMAND[@]}'"
+"${STARTCOMMAND[@]}"
```

Comment:
Had to change `"${STARTCOMMAND[*]}"` to `"${STARTCOMMAND[@]}"` otherwise you get the following error: `./start.sh: line 89: ./PalServer.sh -port=8211 -players=16 -servername="World of Pals" -serverdescription="" -serverpassword="worldofpals" -adminpassword="adminPasswordHere" -queryport=27015 -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS: No such file or directory`

Changed the echo just to match the command.